src/libraries/System.Private.Runtime.InteropServices.JavaScript/src/System/Runtime/InteropServices/JavaScript/Runtime.CS.Owned.cs

```diff
@@ -0,0 +1,146 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections.Generic;
+
+namespace System.Runtime.InteropServices.JavaScript
+{
+    public static partial class Runtime
+    {
+        private static readonly Dictionary<int, WeakReference<JSObject>> _csOwnedObjects = new Dictionary<int, WeakReference<JSObject>>();
+
+        public static JSObject? GetCSOwnedObjectByJsHandle(int jsHandle, int shouldAddInflight)
```

Comment:
```suggestion
        public static JSObject? GetCsOwnedObjectByJsHandle(int jsHandle, int shouldAddInflight)
```

Reply:
That suggestion will make it not compile I guess :-)

Reply:
`CS` and `JS`